app/src/main/java/com/example/next_show/fragments/FeedFragment.java

```diff
@@ -151,41 +168,116 @@ public boolean onNavigationItemSelected(@NonNull MenuItem item) {
         return currView;
     }
 
+    private void resetPage() {
+        // clears the screen and current list of shows
+        adapter.clear();
+
+        // resets filters
+        showFilter.setUpFilters();
+
+        // reset vars
+        selectedGenres = new boolean[]{false, false, false};
+        selectedNetworks = new boolean[]{false, false};
+        selectedYears = new boolean[]{false, false, false};
```

Comment:
Yeahhh, I think having sets would be better here -- then you could just clear the set